### DISCUSSION TITLE
Question on adding button to specific renderer

### DISCUSSION THREAD

**Discussion by jkissi** [marcomusy/vedo#649] [Q&A] [answered]
Hello Marco, 
I have a quick question I hope you can help with. I am trying to add a series of buttons to a specific renderer in a group that is arranged in a similar way to your mulitviewer2 example.  Currently, the button seem to want to add themselves to renderer 1 and I want them to always default to renderer 0 (the background renderer).  Whenever I try to use the at() Plotter function to do this, I get:

```
  File "D:\life\canada\uwo\ece\PhD\research\code\nnse\visualiser\visualisation_ve2.py", line 193, in run
    self.vis, self.pcds = self.addbutton_normals(self.vis, self.pcds)

  File "D:\life\canada\uwo\ece\PhD\research\code\nnse\visualiser\visualisation_ve2.py", line 350, in addbutton_normals
    bu_norm = vis.at(0).addButton(

AttributeError: 'Plotter' object has no attribute 'at'
```

any idea where I am going wrong here?

cheers
j

**Comment by marcomusy** [comment, marked as answer]
I think you must have an old vedo version, try
`pip install vedo -U`
